src/index.html

```diff
@@ -11,7 +11,125 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <h1>HTML Form
+    </h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="section">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input
+            type="text"
+            name="Surname"
```

Comment:
name attribute should be in camelCase. Please fix everywhere

Reply:
Hello, Viktor!
Ok. Changed everything.